kubeflow/openmpi/prototypes/openmpi.jsonnet

```diff
@@ -0,0 +1,21 @@
+// @apiVersion 0.1
+// @name io.ksonnet.pkg.kubeflow-openmpi
+// @description Prototypes for running openmpi jobs.
+// @shortDescription Prototypes for running openmpi jobs.
+// @param name string Name to give to each of the components.
+// @param image string Docker image with openmpi.
+// @param pubkey string Base64-encoded public key used by openmpi.
+// @param prikey string Base64-encoded private key used by openmpi.
+// @optionalParam namespace string null Namespace to use for the components. It is automatically inherited from the environment if not set.
+// @optionalParam workers number 4 Number of workers.
+
+local k = import "k.libsonnet";
+local openmpi = import "kubeflow/openmpi/all.libsonnet";
+
+// updatedParams uses the environment namespace if
+// the namespace parameter is not explicitly set
+local updatedParams = params {
+  namespace: if params.namespace == "null" then env.namespace else params.namespace,
+};
+
+std.prune(k.core.v1.list.new(openmpi.all(updatedParams)))
```

Comment:
I suggest passing params and env both to all and then extracting the namespace their as in
https://github.com/kubeflow/kubeflow/blob/master/kubeflow/new-package-stub/all.libsonnet#L22

Reply:
Done.